app/javascript/src/components/Authentication/Login.jsx

```diff
@@ -44,60 +48,56 @@ const Login = ({ history }) => {
           Sign In
         </h2>
         <Formik
-          initialValues={formInitialValues.loginForm}
+          initialValues={LOGIN_FORM_INITIAL_VALUES}
           validateOnBlur={submitted}
           validateOnChange={submitted}
-          onSubmit={onSubmit}
-          validationSchema={formValidationSchemas.loginForm}
+          onSubmit={handleSubmit}
+          validationSchema={LOGIN_FORM_VALIDATION_SCHEMA}
         >
-          {({ handleSubmit }) => (
-            <Form className="w-full p-8 space-y-6 bg-white border rounded-md shadow">
-              <Input
-                name="email"
-                type="email"
-                placeholder="oliver@example.com"
-                required
-                label="Email"
-                data-cy="login-email-text-field"
-              />
-              <Input
-                name="password"
-                type="password"
-                placeholder="******"
-                required
-                label="Password"
-                data-cy="login-password-text-field"
-              />
-              <Button
-                type="submit"
-                onClick={e => {
-                  e.preventDefault();
-                  setSubmitted(true);
-                  handleSubmit();
-                }}
-                loading={loading}
-                fullWidth
-                className="h-8"
-                label="Login"
-                data-cy="login-submit-button"
-              />
-            </Form>
-          )}
+          <Form className="w-full p-8 space-y-6 bg-white border rounded-md shadow">
+            <Input
+              required
+              name="email"
+              label="Email"
+              type="email"
+              placeholder="oliver@example.com"
+              data-cy="login-email-text-field"
+            />
+            <Input
+              required
+              name="password"
+              label="Password"
+              type="password"
+              placeholder="******"
+              data-cy="login-password-text-field"
+            />
+            <Button
+              fullWidth
+              type="submit"
+              label="Login"
+              data-cy="login-submit-button"
+              className="h-8"
+              loading={loading}
+              onClick={() => {
+                setSubmitted(true);
+              }}
+            />
+          </Form>
         </Formik>
         <div className="flex flex-col items-center justify-center mt-4 space-y-2">
           <div className="flex flex-row items-center justify-start space-x-1">
             <p className="font-normal text-gray-600">{`Don't have an account?`}</p>
```

Comment:
Please remove the template strings. It's an anti-pattern when using it like this in JSX.